tools/force_python_version.py

```diff
@@ -0,0 +1,43 @@
+'''
+Provides a way to run a script on the preferred Python version
+'''
+
+import os
+import sys
+
+'''
+Imports the target script by filename and calls run()
+'''
+def run_by_import(filename, main):
+  import importlib
+  return getattr(importlib.import_module(os.path.basename(filename)), main)()
+
+'''
+Opens Python 2 subprocess to run target script
+'''
+def run_by_subprocess(filename):
+  # Python on Windows does not provide `python2` but instead `py` that receives version parameter
+  py2 = ['py', '-2'] if sys.platform.startswith('win') else ['python2']
+  import subprocess
+  return subprocess.run(py2 + [os.path.realpath(filename) + '.py'] + sys.argv[1:]).returncode
+
+def allowed_version():
```

Comment:
how about `is_allowed_version`, since this checks it, and doesn't return the allowed version?